config/config.yaml

```diff
@@ -0,0 +1,52 @@
+/**:
+  # Global Parameters
+  ros__parameters:
+    use_sim_time: true
+    tf_timeout_threshold: 0.1
+
+  # Basic Robotics Functions
+  platform:
+    ros__parameters:
+      cmd_freq: 50.0  # Hz of platform commands send
+
+  # State Estimator
+  state_estimator:
+    ros__parameters:
+      use_gazebo_tf: true
+      plugin_name: "ground_truth"
+      # Outdoor (GPS)
+      use_gps: true
```

Comment:
same config file for indoor and outdoor? Wouldn't it be indoor config the default one?

Reply:
I think it's clearer for the ROS 2 user to indicate which parameters to modify so they know what's happening. Especially on other platforms where they have to configure them (Crazyflie with the URIs, Tello with the Mocap, etc.). By default, it is set to GPS so that it works with all the mission_*.py scripts in the project.